test/nl/nl_relative.test.ts

```diff
@@ -71,7 +71,7 @@ test("Test - Past relative expressions", () => {
 
 test("Test - Future relative expressions", () => {
     // next hour
-    testSingleCase(chrono.nl, "komende uur", new Date(2016, 10 - 1, 1, 12), (result, text) => {
```

Comment:
Could you add more tests for the version without "-e" instead of changing the current ones?

Reply:
I see. I think we should go with your suggestion.